src/messages/formatters.ts

```diff
@@ -79,6 +81,70 @@ export function blockQuote<C extends string>(content: C): `>>> ${C}` {
 	return `>>> ${content}`;
 }
 
+/**
+ * Formats the URL into `<>`, which stops it from embedding.
+ * @param url The URL to wrap.
+ * @returns The formatted content.
+ */
+export function maskedLink<C extends string>(url: C): `<${C}>`;
+
+/**
+ * Formats the URL into `<>`, which stops it from embedding.
+ * @param url The URL to wrap.
+ * @returns The formatted content.
+ */
+export function maskedLink(url: URL): `<${string}>`;
+export function maskedLink(url: string | URL) {
```

Comment:
I'm not fond of this name, it's actually synonymous for hyperlink (`[]()`).

The alternatives I came up with are:

- `escapedLink`: doesn't escape the link, only it's embedding.
- `hiddenLink`: the link itself is shown, what is not is the embed.
- `detachLink`: opposite of `embedLink`, may work? Just doesn't have a nice ring.
- `removeLinkEmbed`/`removeEmbedLink`: represents what it does, but is long and hard to memorize. Same goes for detach/hide variations.

cc: @discordjs/the-big-3

Reply:
`hideLinkEmbed`, it's technically not removing (the message flag is "removing"). Alternative ~~discord-api-types styled~~ name could be `preventLinkEmbed`

Reply:
I like `preventLinkEmbed`.

Reply:
I would prefer `hideLinkEmbed`

Reply:
Changed in 6991c49.